src/frontend/packages/lib_video/src/components/common/VideoPlayer/index.spec.tsx

```diff
@@ -14,7 +14,9 @@ import {
 } from 'lib-components';
 import { render } from 'lib-tests';
 import React from 'react';
+import { act } from 'react-dom/test-utils';
```

Comment:
should not we use the one from react testing library ?

Reply:
https://github.com/openfun/marsha/pull/2137/commits/d6443b706f9210b4c2e3da436bb9f19a02b0b7fc